docs/generate-model-from-third-party-resources.md

```diff
@@ -23,26 +31,32 @@ don't forget to deploy your CRDs into the local-run kubernetes cluster.
 
 ##### Generate Java model from the downloaded openapi spec
 
-Run the following command and there will be a java sources generated under `/tmp/java`. 
+Run the following commands and there will be a java sources generated under `/tmp/java`. 
 
 ```
-docker run -i --rm yue9944882/java-model-gen < /tmp/swagger | tar -xzf - -C /tmp/
+bash java-crd-cmd.sh -o /tmp/java  < /tmp/swagger 
 ```
 
-By default, the package-name of generated codes will be `io.kubernetes.client`. If you want to override the package-name, consider 
+By default, the java package-name of generated codes will be `io.kubernetes.client`. If you want to override the output package-name, consider 
 use `-p <package_name>` flag. Additionally, you can use `-l <length>` option to make the generator to trim the length of generated 
 class name, the lower number generates shorter names. e.g. for a CRD named `crontabs.v1.example.io`, with `-l 1` the generated 
-name will be `CronTab`, otherwise with `-l3` it will be `ExampleV1Crontab`.
+name will be `CronTab`, otherwise with `-l3` it will be `ExampleV1Crontab`. And also if you feel like only generating models for a few CRDs
+instead of the whole lump of downloaded openapi specs, please use `-n` to filter out your target CRDs. For all supported flags, use `-h` 
+for help info as is shown in the following:
 
 ```
-docker run -i --rm yue9944882/java-model-gen -p com.example < /tmp/swagger | tar -xzf - -C /tmp/
+Usage: generate a java project using input openapi spec from stdin
+ -c: project version of the generated java project.
```

Comment:
What do you mean by the "project version"? Could the option documentation be re-worded for better understanding?

Reply:
removed the line as this looks like a rarely used option 🤔